src/api/pagination.rs

```diff
@@ -13,3 +13,34 @@ pub(super) fn next_cursor(value: &str) -> Option<&str> {
             }
         })
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_pagination_empty_string() {
+        let result = next_cursor("");
+        assert_eq!(result, None);
+    }
+
+    #[test]
+    fn test_pagination_with_next() {
+        let result = next_cursor(
+            "<https://sentry.io/api/0/organizations/sentry/releases/?&cursor=100:-1:1>; \
+            rel=\"previous\"; results=\"false\"; cursor=\"100:-1:1\", \
+            <https://sentry.io/api/0/organizations/sentry/releases/?&cursor=100:1:0>; \
+            rel=\"next\"; results=\"true\"; cursor=\"100:1:0\"",
+        );
+        assert_eq!(result, Some("100:1:0"));
+    }
+
+    #[test]
+    fn test_pagination_without_next() {
+        let result = next_cursor(
+            "<https://sentry.io/api/0/organizations/sentry/releases/?&cursor=100:-1:1>; \
```

Comment:
using a raw string like `r#""#` would be nice because then you won’t have to escape the quotes. Though not sure how that plays with the line continuations.

Reply:
The line continuations would appear as actual newline characters in the string, since the raw string does not allow us to escape anything (including newlines). If we do a raw string, everything needs to be on a single line to ensure the string has the same value. 

I think it's more readable to break the string across multiple lines than to get rid of the escaped quotes but have everything on a single line. So, if it's cool with you, let's keep it as is.